vars/buildPlugin.groovy

```diff
@@ -303,12 +277,7 @@ def call(Map params = [:]) {
                     publishingIncrementals = true
                   } else {
                     String artifacts
```

Comment:
```suggestion
```

Note that you could have accepted https://github.com/jenkins-infra/pipeline-library/pull/691#discussion_r1244343602 (which included this deletion) directly from the GH web UI.

Reply:
> Note that you could have accepted https://github.com/jenkins-infra/pipeline-library/pull/691#discussion_r1244343602 (which included this deletion) directly from the GH web UI.

GH refused to push the comment, because of deleted lines, apparently :/

Reply:
Oh right, GH has problems with suggestions that span edits already in the PR.

Reply:
I miss Review Board.